decidim-core/app/types/decidim/core/user_group_type.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Core
+    # This type represents a UserGroup
+    UserGroupType = GraphQL::ObjectType.define do
+      name "UserGroup"
+      description "A user group"
+
+      interfaces [
+        -> { Decidim::Core::AuthorInterface }
+      ]
+
+      field :id, !types.ID, "The user group's id"
+
+      field :name, !types.String, "The user group's name"
+
+      field :nickname, !types.String, "User groups have no nickname" do
```

Comment:
🤷‍♂️

Reply:
Don't really know what's going on with this xD. We should probably revisit the comments API at some point.